conan/api/subapi/upload.py

```diff
@@ -101,16 +101,16 @@ def _upload_pkglist(pkglist, subtitle=lambda _: None):
         elapsed = time.time() - t
         ConanOutput().success(f"Upload completed in {int(elapsed)}s\n")
 
-    def get_backup_sources(self, package_list=None):
+    def get_backup_sources(self, package_list=None, exclude=True, only_upload=True):
         """Get list of backup source files currently present in the cache,
         either all of them if no argument, else filter by those belonging to the references in the package_list"""
         config = self.conan_api.config.global_conf
         download_cache_path = config.get("core.sources:download_cache")
         download_cache_path = download_cache_path or HomePaths(
             self.conan_api.cache_folder).default_sources_backup_folder
-        excluded_urls = config.get("core.sources:exclude_urls", check_type=list, default=[])
+        excluded_urls = config.get("core.sources:exclude_urls", check_type=list, default=[]) if exclude else []
```

Comment:
What happen if you let it filter by exclude_urls? Isn't the result the same?

Reply:
You would not be able to remove backups made pf urls in your exclusion list, those are explicitly filtered out

Reply:
Will add a test for excluded urls

Reply:
But if they are excluded, they are not backed-up? Or adding a exclusion after it has been backed-up?

Reply:
I see, the local caching happens even if it is excluded to be backuped in the server again. Understood.